### DISCUSSION TITLE
Message: "An error occurred" on 200 Status Log Entries

### DISCUSSION THREAD

**Discussion by jadamconnor** [vercel/community#245] [Help] [answered]
Since last night at about 18:00 central time, all of my logs that aren't lambdas are coming with a message "An error occurred." The vast majority of these are 200 status, but it doesn't seem to matter whether its a 4xx either. All logs come with this non-specific warning. I've attempted to find the message in the responses in browser in the network tab, but they all look normal. Anyone know if this might be related to Vercel? Example below:

```
{
  "id": "xxx",
  "message": ": An error occurred",
  "requestId": "xxx",
  "proxy": {
    "timestamp": 1643134272023,
    "region": "gru1",
    "errorCode": "",
    "method": "GET",
    "statusCode": 200,
    "referer": "https://smarthomestarter.com/how-many-years-are-covered-under-warranty-for-a-hisense-tv/",
    "path": "/favicon.ico",
    "cacheId": "-",
    "host": "smarthomestarter.com",
    "scheme": "https",
    "clientIp": "xxx",
    "userAgent": [
      "Mozilla/5.0 (iPhone; CPU iPhone OS 15_1_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.1 Mobile/15E148 Safari/604.1"
    ]
  },
  "projectId": "xxx",
  "deploymentId": "xxx",
  "source": "static",
  "host": "xxx",
  "path": "favicon.ico",
  "@timestamp": "2022-01-25T18:11:12.027Z",
  "@timestamp_received": "2022-01-25T18:11:18.788Z",
  "logsene_orig_type": "logs"
}
```

**Comment by JTaylor0196** [comment]
Hey there, thanks for your input! As a result of this feedback, the team is looking into this and we'll let you know what we find. Stay tuned!

**Comment by JTaylor0196** [comment, marked as answer]
Hey there @jadamconnor! Circling back to update you and let you know that the team created and rolled out a fix for this issue today. Please let me know if you still happen to experience this issue, but, moving forward, you should be all set.

Thanks for your patience while we've worked through this!

**Reply by jadamconnor** [reply]
Hey, thanks, sorry for the delayed response. I was in an email thread with Vercel about this as well as with my logging provider. Thanks again for getting back to me and glad I could help. Happy to report the issue is resolved.
